plugins/slack_app/app/models/slack_identifier.rb

```diff
@@ -0,0 +1,8 @@
+class SlackIdentifier < ActiveRecord::Base
+  belongs_to :user
+
+  def self.app_token
+    record = SlackIdentifier.find_by_user_id nil
```

Comment:
Yeah, this is a bit of a hack. We need a Slack API token in order to post chat messages, and I'm storing that in this table with `user_id = nil`. I'd like this to be smooth when the Slack admin authorizes the app to the team, and not stop the whole process until she can get an ops person to change the server configuration to store this value. Is there a better way?